Tools/Scripts/webkitdirs.pm

```diff
@@ -1227,8 +1227,9 @@ sub XcodeOptions
     push @options, @baseProductDirOption;
     push @options, "ARCHS=$architecture" if $architecture;
     push @options, "SDKROOT=$xcodeSDK" if $xcodeSDK;
-    if (xcodeVersion() lt "15.0") {
-        push @options, "TAPI_USE_SRCROOT=YES" if $ENV{UseSRCROOTSupportForTAPI};
+    if (xcodeVersion() ge "14.0" && xcodeVersion() lt "15.0") {
```

Comment:
Is checking for Xcode version right here, or do we need to check toolchain?

Reply:
Xcode alone should be fine – what matters here is whether XCBuild is new enough to recognize `TAPI_USE_SRCROOT` without the defaults flag flipped.